src/components/IconsPreview/Icons.module.css

```
@@ -31,6 +31,18 @@
 }
 
 .Icon__Tile {
+  border: none;
```

Stylelint (17.14.0)

[error] Duplicate property "border" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)